src/app.ts

```diff
@@ -98,8 +99,9 @@ function withSpidAuth(
     next: express.NextFunction
   ) => {
     passport.authenticate("spid", async (err, user) => {
+      const issuer = getSamlIssuer(req.body);
       if (err) {
-        log.error("Error in SPID authentication: %s", err);
+        log.error("Error in SPID authentication: %s | Issuer: %s", err, issuer);
```

Comment:
we are using the following format in other components, since we're changing the code already, can you make it like that?

```
SpidStrategy|Authentication error|ERROR=${err}|ISSUER=${issuer}
```

(_SpidStrategy_ is the functions/component that is logging, feel free to use a different name)

Reply:
fixed in d098c6c